src/app/malaria-101/malaria-101.component.scss

```diff
@@ -0,0 +1,238 @@
+@import "../styles/main";
+
+video {
```

Comment:
Yes, this is the same file as the one in my other open PR.
It has additional css for the new activity